tasks/README.md

```diff
@@ -6,4 +6,71 @@ This directory contains the tasks that are part of this benchmark.
 Name | Summary | Category
 ---- | ----------- | --------
 `task1001_squad1.1_question_generation` | Generating guestions (based on SQuAD 1.1) | Question Generation  
-`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
\ No newline at end of file
+`task1002_squad1.1_answer_generation` | Generating answers to SQuAD 1.1 questions | Answer Generation
+`task417_menyo20k-mt_en_yo_translation` | Given an English sentence convert it to Yoruba. | Translation
+`task418_menyo20k-mt_yo_en_translation` | Given a sentence in Yoruba convert it to English. | Translation
```

Comment:
Please add variants for this task and other tasks which do not have variants in this PR

Reply:
Hi,
The tasks without variants are the ones given as initial homework. For the project the tasks assigned to me were **189-211**. Variants will be present only for these tasks. Please check them.